projects/packages/my-jetpack/src/class-rest-products.php

```diff
@@ -83,6 +83,27 @@ public function __construct() {
 				),
 			)
 		);
+
+		register_rest_route(
+			'my-jetpack/v1',
+			'site/products/ownership',
```

Comment:
As mentioned in DM, `site/products/ownership` falls unfortunately into `site/products/(?P<product>[a-z\-]+)` wildcard defined above. We need to either move that definition above, or change the endpoint path to something else (imo, this solution is better as it doesn't depend on sequencing - but feel free to choose, both are good)